dcc-portal-ui/app/scripts/ui/js/table.js

```diff
@@ -238,6 +244,24 @@ angular.module('icgc.ui.table.pagination', [])
         };
       }
     };
+  })
+  .directive('paginationClientSide', function(){
```

Comment:
Seems I might be missing context
I guess we have a separate directive for things that are paginated server side?
Would it be possible to coalesce those two?